source/16-6-2_Outcomes.markdown

```diff
@@ -70,15 +104,73 @@ If you have more than one measure to add to a category, you can also opt to put
 - Click on 'bulk add new {{outcome}} measures.
 - Enter the measures in the text box, each one separated by a semi-colon. If you choose to add them in this way, you'll need to find each one in the list and edit it (click on the pen and paper icon to the right to open the editing box) to include the type, values and notes, as we saw with the single measure before. 
 
-[Editing Icon](16.6.2f.png)
+![Editing Icon](16.6.2f.png)
 
 - Continue these steps until you have added all the measures that you need.
 
+#### Scores with user specified issue
+
+This measure type is intended to be used where you want to record a score, but also want to record what the issue is.
+One option is to create a set of different measures, one for each type of issue, and recording the detail in the 
+comments box. However this can feel prescriptive and may not be appropriate for your situation. It can also lead to 
+a long list that becomes hard to navigate.
+
+The "scores with user specified issue" measure type allows you to record the score and the issue in one place. The 
+first time you enter data for a particular {{user}} you will be prompted to enter the issue. The next time you enter
+data for that {{user}} you will see the previously entered issue and can just update the score.
```

Comment:
Repetition of lines 75,75,77

Reply:
Yes but I think it's necessary here for completeness, and will mean the rest of this section is harder to read / understand?